index.js

```diff
@@ -271,15 +271,20 @@ export default class QRCodeScanner extends Component {
 			if (this.props.cameraTimeout > 0) {
 				this.timer && clearTimeout(this.timer);
 				this.timer = setTimeout(() => this._setCamera(false), this.props.cameraTimeout);
-			}
+      }
+
+      let height = styles.camera.height;
+      if (this.props.cameraStyle && this.props.cameraStyle.height) {
+        height = this.props.cameraStyle.height;
+      }
 
 			if (this.props.fadeIn) {
 				return (
 					<Animated.View
 						style={{
 							opacity: this.state.fadeInOpacity,
 							backgroundColor: 'transparent',
-							height: styles.camera.height,
+							height,
```

Comment:
try to consider a simpler solution for this problem like this one:

https://github.com/jkrecek/react-native-qrcode-scanner/blob/27e38e17f023e69b403c4141b0a3d84a6a6e044f/index.js#L282

Reply:
done, when will a new version be created?